example/android/javalib/1-hello-world/build.mill

```diff
@@ -108,7 +108,7 @@ object app extends AndroidAppModule {
 > ./mill show app.test
 ...compiling 2 Java source...
 
-> cat out/app/test/testForked.dest/out.json
+> cat out/app/test/testForked.dest/worker-0/out.json
```

Comment:
My question is, how did it work in `main`? It maybe passed due to selective tests or something?

Reply:
Most likely selective tests. I saw it fail very often too, mostly in PR run.

Reply:
I think they were broken, and @alexarchambault  disabled them by adding a `ignoreErrorsOnCI` marker file. Could you try removing the file in this PR and see if they pass?

Reply:
The timeout I see in the last run is quite common, last time I was trying to fix it , my hypothesis was running the emulator task in parallel with other tasks was causing slowness.

We could have a simpler test for the java example, without the emulator and keep only the kotlin one, or try to force the android tasks to run sequentially again

Reply:
Removed the markers, build is green.